packages/@aws-cdk/aws-stepfunctions-tasks/lib/codebuild/start-build.ts

```diff
@@ -12,12 +12,187 @@ export interface CodeBuildStartBuildProps extends sfn.TaskStateBaseProps {
    * CodeBuild project to start
    */
   readonly project: codebuild.IProject;
+  /**
+   * Build output artifact settings that override, for this build only.
+   *
+   * @default - the latest ones already defined in the build project.
```

Comment:
I'm assuming this is retrieved from https://docs.aws.amazon.com/codebuild/latest/APIReference/API_StartBuild.html#API_StartBuild_RequestParameters

the abbreviation used in the comment here seems off and the text for the default should be a part of the doc string. The default behaviour should be the same as in some of the other properties: `i.e. the artifacts specified in the build project`